docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -35,17 +35,17 @@ The main features of the Sensor API include:
-  | [Geomagnetic Rotation Vector Sensor](#mag_rotation) | [Humidity Sensor](#humidity)             | [Rotation Vector Sensor](#rotation)      |
-  | [Gravity Sensor](#gravity)               | [Light Sensor](#light)                   | [Significant Motion Sensor](#significant_motion) |
-  | [Gyroscope](#gyro)                       | [Linear Acceleration Sensor](#lin_accelerometer) | [Sleep Monitor](#sleep_monitor)          |
-  | [Gyroscope Rotation Vector Sensor](#gyro_rotation) | [Magnetic Sensor](#magnetic)             | [Temperature Sensor](#temperature)       |
-  | [Heart Rate Monitor LED Green Sensor](#hrm_green) | [Orientation Sensor](#orientation)       | [Ultraviolet Sensor](#ultraviolet)       |
-  | [Heart Rate Monitor LED IR Sensor](#hrm_ir) | [Pedometer](#pedometer)                  | [Uncalibrated Gyroscope](#uncal_gyro)    |
-  | [Heart Rate Monitor LED Red Sensor](#hrm_red) | [Pressure Sensor](#pressure)             | [Uncalibrated Magnetic Sensor](#uncal_magnetic) |
-
+  | Supported sensor types                                        |                                                  |                                                  |
+  |---------------------------------------------------------------|--------------------------------------------------|--------------------------------------------------|
+  | [Accelerometer](#accelerometer)                               | [Heart Rate Monitor LED IR Sensor](#hrm_ir)      | [Proximity Sensor](#proximity)                   |
+  | [Geomagnetic Rotation Vector Sensor](#mag_rotation)           | [Heart Rate Monitor LED Red Sensor](#hrm_red)    | [Rotation Vector Sensor](#rotation)              |
+  | [Gravity Sensor](#gravity)                                    | [Humidity Sensor](#humidity)                     | [Significant Motion Sensor](#significant_motion) |
+  | [Gyroscope](#gyro)                                            | [Light Sensor](#light)                           | [Sleep Monitor](#sleep_monitor)                  |
+  | [Gyroscope Rotation Vector Sensor](#gyro_rotation)            | [Linear Acceleration Sensor](#lin_accelerometer) | [Temperature Sensor](#temperature)               |
+  | [Heart Rate Monitor Sensor](#hrm)                             | [Magnetic Sensor](#magnetic)                     | [Ultraviolet Sensor](#ultraviolet)               |
+  | [Heart Rate Monitor Batch Sensor](#hrm_batch)                 | [Orientation Sensor](#orientation)               | [Uncalibrated Gyroscope](#uncal_gyro)            |
+  | [Heart Rate Monitor LED Green Sensor](#hrm_green)             | [Pedometer](#pedometer)                          | [Uncalibrated Magnetic Sensor](#uncal_magnetic)  |
+  | [Heart Rate Monitor LED Green Batch Sensor](#hrm_green_batch) | [Pressure Sensor](#pressure)                     |                                                  |
```

Comment:
@mijin85cho As I have requested for heading style change from title to sentence, should we also change the case of all sensor names in this table too? Request your opinion on this? 

@bbrto21 what is your suggestion on this?

Reply:
@nawab-samsung I agree with you

Reply:
@nawab-samsung As the titles are changed to follow the sentence style, I also agree to match the names in the table too.

Reply:
@nawab-samsung pleae review again

Reply:
Thanks @mijin85cho & @bbrto21